iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/execution/config/metadata/ShowRegionTask.java

```diff
@@ -111,6 +112,19 @@ public static void buildTSBlock(
                 new Binary(
                     DateTimeUtils.convertLongToDate(regionInfo.getCreateTime()),
                     TSFileConfig.STRING_CHARSET));
+        // region size
+        String regionSizeStr = "";
+        if (regionInfo.getConsensusGroupId().getType().ordinal()
+            == TConsensusGroupType.DataRegion.ordinal()) {
+          if (regionInfo.getTsfileSize() != -1) {
+            regionSizeStr = FileUtils.humanReadableByteCountSI(regionInfo.getTsfileSize());
+          } else {
+            regionSizeStr = "Unknown";
+          }
+        }
+        builder
+            .getColumnBuilder(12)
+            .writeBinary(new Binary(regionSizeStr, TSFileConfig.STRING_CHARSET));
```

Comment:
whether can use BytesUtils.valueOf() ?            change string to byte

Reply:
good point